Makefile

```diff
@@ -217,6 +228,25 @@ close-test:
 	@echo "+\n++ Make: Closing test container ...\n+"
 	@docker-compose -f docker-compose.test.yml down
 
+# ======================================================================
+# Local Development Environment & Testing
+# ======================================================================
+
+build:
+	@docker-compose up --build -d --force-recreate
+
+start: 
+	docker-compose up -d 
+
+stop: 
+	@docker-compose down
+
+wipe: 
+	@docker-compose down -v --remove-orphans
+
+be-logs:
+	@docker logs $(PROJECT)-backend --follow --tail 25
```

Comment:
I think we can remove the `make local-backend-logs` command now as we have this one :)

Reply:
it is removed.